<ui version="4.0" >
 <class>ControlsWidget</class>
 <widget class="QWidget" name="ControlsWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>33</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Spatial Plots Controls</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="leftMargin" >
    <number>2</number>
   </property>
   <property name="topMargin" >
    <number>2</number>
   </property>
   <property name="rightMargin" >
    <number>2</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <property name="spacing" >
      <number>9</number>
     </property>
     <item>
      <widget class="QPushButton" name="playPauseButton" >
       <property name="text" >
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="positionSlider" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox" />
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>positionSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel" >
     <x>319</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>positionSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>329</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel" >
     <x>129</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
